packages/expo-crypto/README.md

```diff
@@ -0,0 +1,228 @@
+You can specify the returned string format as one of `CryptoEncoding`. By default the resolved value will be formatted as a `HEX` string.
+
+| 🍎 iOS | 💚 Android | 💻 Web |
+| ------ | ---------- | ------ |
+| ✅     | ✅         | ✅     |
+
+**Parameters**
+
+| Name      | Type                                      | Description                                                                         |
+| --------- | ----------------------------------------- | ----------------------------------------------------------------------------------- |
+| algorithm | [`CryptoDigestAlgorithm`][algorithm-link] | Transforms a value into a fixed-size hash (usually shorter than the initial value). |
+| data      | `string`                                  | The value that will be used to generate a digest.                                   |
+| options   | `CryptoDigestOptions`                     | Format of the digest string. Defaults to: `CryptoDigestOptions.HEX`                 |
+
+**Returns**
+
+| Name   | Type              | Description                                          |
+| ------ | ----------------- | ---------------------------------------------------- |
+| digest | `Promise<string>` | Resolves into a value representing the hashed input. |
+
+**Example**
+
+```ts
+const digest = await Crypto.digestStringAsync(
+  Crypto.CryptoDigestAlgorithm.SHA.SHA512,
+  '🥓 Easy to Digest! 💙'
+);
+```
+
+## Types
+
+### `CryptoDigestAlgorithm`
+
+[`Cryptographic hash function`][algorithm-link] is an algorithm that can be used to generate a checksum value. They have a variety of applications in cryptography.
+
+> Cryptographic hash functions like `SHA1`, `MD5` are **vulnerable**! Attacks have been proven to significantly reduce their collision resistance.
+
+| Name              | Type   | Description | Collision Resistant | 🍎 iOS | 💚 Android | 💻 Web |
+| ----------------- | ------ | ----------- | ------------------- | ------ | ---------- | ------ |
+| [SHA1][sha-def]   | string | `160` bits  | ❌                  | ✅     | ✅         | ✅     |
+| [SHA256][sha-def] | string | `256` bits  | ✅                  | ✅     | ✅         | ✅     |
+| [SHA384][sha-def] | string | `384` bits  | ✅                  | ✅     | ✅         | ✅     |
+| [SHA512][sha-def] | string | `512` bits  | ✅                  | ✅     | ✅         | ✅     |
+| MD2               | string | `128` bits  | ❌                  | ✅     | ✅         | ❌     |
+| MD4               | string | `128` bits  | ❌                  | ✅     | ✅         | ❌     |
+| MD5               | string | `128` bits  | ❌                  | ✅     | ✅         | ❌     |
+
+### `CryptoEncoding`
+
+| Name   | Type       | 🍎 iOS | 💚 Android | 💻 Web |
+| ------ | ---------- | ------ | ---------- | ------ |
+| HEX    | `'hex'`    | ✅     | ✅         | ✅     |
+| Base64 | `'base64'` | ✅     | ✅         | ✅     |
+
```

Comment:
Add a note saying that the Base64 encoding has trailing padding and does not wrap lines nor has a trailing newline.